system/firewalld.py

```diff
@@ -47,6 +47,11 @@
     required: false
     default: null
     version_added: "2.0"
+  interface:
+    description:
+      - 'The interface you would like to add/remove to/from a zone in firewalld - zone must be specified'
+    required: false
+    default: null
```

Comment:
When I was thinking about that, I came to the point that the version might be added by the maintainer on release. Is "2.0" the version that should be used here?